packages/tempo/doc/tempo.shorthand.md

```
@@ -0,0 +1,130 @@
+
+---
+
+## 6. Development Constraints (Range-Keys)
+
+When defining ranges in a Terminology Plugin, you must adhere to the **Golden Rules of Range-Keys** to avoid lexer collisions with modifiers.
+
+> [!CAUTION]
+> **Reserved Characters**: Range-Keys **MUST NOT** contain any of the following characters:
+> `> < + = , . ! @ # $ % ^ & * ( ) [ ] { }`
+> These characters are reserved for directional shifters and lexer boundaries.
+
```

markdownlint-cli2 (0.22.0)

[warning] Blank line inside blockquote

(MD028, no-blanks-blockquote)